q2_diversity/_ordination.py

```diff
@@ -28,3 +31,57 @@ def pcoa(distance_matrix: skbio.DistanceMatrix,
 def pcoa_biplot(pcoa: skbio.OrdinationResults,
                 features: pd.DataFrame) -> skbio.OrdinationResults:
     return skbio.stats.ordination.pcoa_biplot(pcoa, features)
+
+
+def tsne(distance_matrix: skbio.DistanceMatrix,
+         number_of_dimensions: int = 3) -> skbio.OrdinationResults:
+
+    data = distance_matrix.data
+    ids = distance_matrix.ids
+
+    if number_of_dimensions == 2:
+        number_of_dimensions = 3
+        tsne = TSNE(2).fit_transform(data)
+        add_zeros = np.zeros((tsne.shape[0], 1), dtype=np.int64)
+        tsneData = np.append(tsne, add_zeros, axis=1)
+
+    else:
+        tsneData = TSNE(number_of_dimensions).fit_transform(data)
```

Comment:
can you use a named parameter here for clarity, i.e. `TSNE(n_components=number_of_dimension)`?